src/prisma/schema.prisma

```diff
@@ -2086,3 +2088,17 @@ model CharmTransaction {
   toWallet   CharmWallet? @relation("toWallet", fields: [to], references: [id], onDelete: SetNull)
   metadata   Json?        @default("{}")
 }
+
+model FavoriteCredential {
+  id                   String            @id @default(uuid()) @db.Uuid
+  index                Int               @default(-1)
+  chainId              Int?
+  userId               String            @db.Uuid
+  user                 User              @relation(fields: [userId], references: [id], onDelete: Cascade)
+  attestationId        String?
+  issuedCredentialId   String?           @db.Uuid
+  issuedCredential     IssuedCredential? @relation(fields: [issuedCredentialId], references: [id], onDelete: SetNull)
+  gitcoinWalletAddress String?
```

Comment:
Please double check what format we send to the Gitcoin API (lowercase or checksummed) and make sure it's compatible with what's stored here

If lowercase works on the API, let's go with that here to align with rest of the system

Reply:
Yep it does work, I will store the lowercased version on db